stressberry/cli.py

```diff
@@ -75,6 +96,8 @@ def run(argv=None):
     parser = _get_parser_run()
     args = parser.parse_args(argv)
 
+    use_vcgencmd = vcgencmd_avaialble() and not args.disable_vcgencmd
```

Comment:
Typo in variable name, but might not be needed after all.

Reply:
If someone provides a temperature or frequency file via the command line. Do we use that file or ignore it and continue to use vcgencmd?

If we want to use file if specified, then is it possible to detect if the arg is passed or not? Since it has a default? And when calling main:measure_temp() if filename isn't provided it again has a default value. 

I'd welcome some guidance around how you think this should be handled.

Reply:
My idea was to remove the file options on the command line. We have `vcgencmd`, so no need for that, right?

Reply:
Yes but you currently have `--frequency-file` and `----temperature-file` Are you suggesting removing use of both those and only ever use vcgencmd? I saw in your original code comment that you'd wanted to use the file method for temperatures over vcgencmd? 

It could just use vcgencmd if available otherwise fall back to file and then use the default which can also be overridden by the cmd line args.  But if vcgencmd is present, then only ever use that method.

Reply:
> It could just use vcgencmd if available otherwise fall back to file and then use the default which can also be overridden by the cmd line args. But if vcgencmd is present, then only ever use that method.

Using files has two advantages:
   * You don't have to call a potentially harmfull cmd tool
   * stressberry could work on machines other than raspberry

However, I don't like the complication on the command line à la "use this temperature file, but only if that other options is also set".

Perhaps we can do it like that: Offer the file cli options, defaulting to `None`. If they are set to a path, use that file, if not, use `vcgencmd`.

Reply:
I think having the option to force the use of a file source is useful, but agree my initial implementation is inelegant (at best). Defaulting to `None` in both `cli.py` and `main.py` would be my preference. I'll work on that implementation and see how it looks.